streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -124,6 +124,7 @@ public void openDB(ProcessorContext context) {
         options.setCreateIfMissing(true);
         options.setErrorIfExists(false);
         options.setInfoLogLevel(InfoLogLevel.ERROR_LEVEL);
+        options.setIncreaseParallelism(Runtime.getRuntime().availableProcessors());
```

Comment:
@dguy The tuning guides I was able to find all recommend calling this function. However, increasing the number of background flushes seems inconclusive. Some are recommending it should be done for bulk load (https://github.com/facebook/rocksdb/issues/490). We have a separate JIRA for bulk load: https://issues.apache.org/jira/browse/KAFKA-4868. Thanks.